ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionIntygetAvser.java

```diff
@@ -0,0 +1,169 @@
+                    .list(
+                        List.of(
+                            CheckboxMultipleCode.builder()
+                                .id(IntygAvserKategori.IAV1.name())
+                                .label(texts.get(INTYG_AVSER_IAV1_LABEL_ID))
+                                .build(),
+                            CheckboxMultipleCode.builder()
+                                .id(IntygAvserKategori.IAV2.name())
+                                .label(texts.get(INTYG_AVSER_IAV2_LABEL_ID))
+                                .build(),
+                            CheckboxMultipleCode.builder()
+                                .id(IntygAvserKategori.IAV3.name())
+                                .label(texts.get(INTYG_AVSER_IAV3_LABEL_ID))
+                                .build(),
+                            CheckboxMultipleCode.builder()
+                                .id(IntygAvserKategori.IAV4.name())
+                                .label(texts.get(INTYG_AVSER_IAV4_LABEL_ID))
+                                .build(),
+                            CheckboxMultipleCode.builder()
+                                .id(IntygAvserKategori.IAV5.name())
+                                .label(texts.get(INTYG_AVSER_IAV5_LABEL_ID))
+                                .build(),
+                            CheckboxMultipleCode.builder()
+                                .id(IntygAvserKategori.IAV6.name())
+                                .label(texts.get(INTYG_AVSER_IAV6_LABEL_ID))
+                                .build(),
+                            CheckboxMultipleCode.builder()
+                                .id(IntygAvserKategori.IAV7.name())
+                                .label(texts.get(INTYG_AVSER_IAV7_LABEL_ID))
+                                .build(),
+                            CheckboxMultipleCode.builder()
+                                .id(IntygAvserKategori.IAV8.name())
+                                .label(texts.get(INTYG_AVSER_IAV8_LABEL_ID))
+                                .build(),
+                            CheckboxMultipleCode.builder()
+                                .id(IntygAvserKategori.IAV9.name())
+                                .label(texts.get(INTYG_AVSER_IAV9_LABEL_ID))
+                                .build(),
+                            CheckboxMultipleCode.builder()
+                                .id(IntygAvserKategori.IAV10.name())
+                                .label(texts.get(INTYG_AVSER_IAV10_LABEL_ID))
+                                .build()
+                        )
+                    )
+                    .build()
+            )
+            .value(
+                CertificateDataValueCodeList.builder()
+                    .list(
+                        intygAvserKategori != null ? intygAvserKategori.stream()
+                            .map(QuestionIntygetAvser::getValueCode)
+                            .collect(Collectors.toList()) : List.of()
```

Comment:
I would probably use Collections.emptyList() instead of List.of(). It is more clear to read... but if you leave it like this I will still approve :-)